src/table/utils/handle-click.ts

```diff
@@ -31,6 +31,18 @@ export const handleMouseDownLabelToFocusHeadCell = (evt: MouseEvent, rootElement
   updateFocus({ focusType: 'focus', cell: getCellElement(rootElement, [0, columnIndex]) });
 };
 
+export const handleClickToSort = (
```

Comment:
no need to put it here. My point was to pit it ouside the elements, like we do for `handleKeyDown`